patches/chrome-browser-download-android-java-src-org-chromium-chrome-browser-download-DownloadManagerBridge.java.patch

```diff
@@ -0,0 +1,40 @@
+diff --git a/chrome/browser/download/android/java/src/org/chromium/chrome/browser/download/DownloadManagerBridge.java b/chrome/browser/download/android/java/src/org/chromium/chrome/browser/download/DownloadManagerBridge.java
+index 9dee29045bb7d38473529f3dd362e279bb53e692..dc1b84f7ef31e4b65dad9d9b00173a968af7d0cc 100644
+--- a/chrome/browser/download/android/java/src/org/chromium/chrome/browser/download/DownloadManagerBridge.java
++++ b/chrome/browser/download/android/java/src/org/chromium/chrome/browser/download/DownloadManagerBridge.java
```

Comment:
Will be included in M87: https://chromium.googlesource.com/chromium/src.git/+/6fcac9a226e802cc99e31c025efc41c40f2512c1

Without this patch downloads to external SD card crash.